### DISCUSSION TITLE
Switch between white mode / black mode

### DISCUSSION THREAD

**Discussion by ghost** [Nain57/Smart-AutoClicker#888] [General Feature Requests]
### Describe the feature

It would be a single icon button to switch between white mode and dark mode in the main window.

### Describe the usage

You may think this requested feature is unnecessary as its for an app you don't need to look on that long. But almost every app has this option (even small ones) and at least on old devices like mine you cannot change this system-wide.
I would like to see an icon at the top right of the screen like in WhatsDeleted (Screenshots) to switch between black and white mode (the setting should also be saved).

Thank you very much for your time and effort you're putting into this app. I really appreciate your work. Keep going!

![ResizedImage_2024-11-21_22-48-46_2516](https://github.com/user-attachments/assets/64c7f6ed-7c91-4584-b7ce-f89b284ca463)
![ResizedImage_2024-11-21_22-47-05_4542](https://github.com/user-attachments/assets/95746b25-5fd7-4fce-a671-6b88e4cc5508)
![ResizedImage_2024-11-21_22-44-35_1329](https://github.com/user-attachments/assets/16c1fdbc-aa65-4094-b3ec-1434b746edd8)

### Is it a duplicate?

- [X] There is no duplicates

**Comment by vibhor1102** [comment]
Not sure if it was broken earlier, but currently Klick'r correctly switches between dark and light modes based on your device settings
